packages/build/tests/deploy/tests.js

```diff
@@ -50,7 +50,8 @@ test('Deploy plugin response syntax error', async (t) => {
   const { address, stopServer } = await startDeployServer({ response: 'test' })
   try {
     const output = await new Fixture('./fixtures/empty').withFlags({ buildbotServerSocket: address }).runWithBuild()
-    t.snapshot(normalizeOutput(output))
+    // This shape of this error can change with different Node.js versions.
+    t.true(output.includes('Invalid response from buildbot: SyntaxError: Unexpected token'))
```

Comment:
You're using a different string here to compare. Will this work across all versions?

Reply:
Yeah I tried grabbing what both options have in common:

```
  Error message␊
  -   Invalid response from buildbot: SyntaxError: Unexpected token 'e', "test" is not valid JSON␊
  +   Invalid response from buildbot: SyntaxError: Unexpected token e in JSON at position 1␊
```